sapi/fpm/fpm/fpm_main.c

```diff
@@ -1836,6 +1836,12 @@ consult the installation file that came with this distribution, or visit \n\
 			/* check if request_method has been sent.
 			 * if not, it's certainly not an HTTP over fcgi request */
 			if (!SG(request_info).request_method) {
+				zend_try {
+					zlog(ZLOG_ERROR, "SCRIPT_FILENAME env not found in cgi env");
```

Comment:
> you should be able to find the info from access log, no needs log here.

no no no...access log will write less info into log aboout time ,script, http result status. But the bug is that it can't write errors info for system administrator. access log is useless.



 >I am not sure, but in fpm's side, it's bad request

yep,that is bad request for fpm's side. but this request is return to browser side. 
it is server's fault for browser.so I think 500 is suitable request http status.